cookbooks/bcpc-hadoop/templates/default/graphite.query_graphite.py.erb

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+import urllib
+import urllib2
+import json
+import sys
+import logging
+from logging.handlers import RotatingFileHandler
+
+LOGGER = None
+
+def create_log(path):
+  """  
+  Creates a rotating log
+  """
+  global LOGGER
+  LOGGER = logging.getLogger('query_graphite')
+  LOGGER.setLevel(logging.<%= node[:bcpc][:hadoop][:zabbix][:query_graphite][:logging_level] %>)
```

Comment:
Kiran, do you suppose you could pull these 'node' object references into local variables on the template resource?

e.g.
```ruby
template '/usr/local/bin/thing.py' do
  variables({:log_level => node[:bcpc][:hadoop][:zabbix][:query_graphite][:logging_level]})
end
```